ias_zone.cpp

```diff
@@ -83,6 +83,12 @@ void DeRestPluginPrivate::handleIasZoneClusterIndication(const deCONZ::ApsDataIn
             return; // sanity
         }
     }
+    
+    // Allow clearing the alarm bit for Develco devices
+    if (!(zclFrame.frameControl() & deCONZ::ZclFCDisableDefaultResponse))
+    {
+        sendZclDefaultResponse(ind, zclFrame, deCONZ::ZclSuccessStatus);
+    }
```

Comment:
Tricky, the default response should only be send when no other response is sent.

Reply:
Hello. Not sure if this is the correct place to comment. Just saw this update and tried it out and it works great. I can stop the siren with the button on the smoke detector and the alarm bit gets cleared. I can also stop the siren from the API and the alarm bit also gets cleared there. Thank you @SwoopX for finding the solution.